subsys/tracing/sysview/sysview.c

```diff
@@ -105,6 +105,35 @@ void sys_trace_mutex_unlock(struct k_mutex *mutex)
 		SYS_TRACE_ID_MUTEX_UNLOCK, (uint32_t)(uintptr_t)mutex);
 }
 
+static void set_thread_name(char *name, struct k_thread *thread)
+{
+	const char *tname = k_thread_name_get(thread);
+
+	if (tname != NULL && tname[0] != '\0') {
+		memcpy(name, tname, sizeof(name));
```

Comment:
The use of `sizeof(name)` looks wrong in this function. Isn't it always 4 on a 32 bit platform? I would expect that THREAD_NAME_LEN is used instead of this instead.

Reply:
Thanks for pointing out this copy-paste mistake